src/Input.h

```diff
@@ -245,6 +248,7 @@ class Input::Manager {
 
 	bool joystickEnabled;
 	bool mouseYInvert;
+	bool noMiddleMouseButton;
```

Comment:
I adopted to surrounding code here but shouldn't they be member variables, m_mouseYInvert, m_noMiddleMouseButton... etc?

Reply:
Technically yes, they're this way for legacy reasons. Let me review the PR further before I make a decision as to whether it's worth converting them.